addons/web/static/src/js/views/pivot/pivot_view.js

```diff
@@ -76,19 +78,22 @@ var PivotView = AbstractView.extend({
             // the measure should be allowed.  However, be careful if you define
             // a measure in your pivot view: non stored functional fields will
             // probably not work (their aggregate will always be 0).
-
-            if (field.attrs.type === 'measure' && !(field.attrs.name in measures)) {
-                measures[field.attrs.name] = self.fields[field.attrs.name];
-            }
-            if (field.attrs.type === 'measure' || 'operator' in field.attrs) {
-                activeMeasures.push(name);
-                measures[name] = self.fields[name];
-            }
-            if (field.attrs.type === 'col') {
-                colGroupBys.push(name);
-            }
-            if (field.attrs.type === 'row') {
-                rowGroupBys.push(name);
+            if (!!field.attrs.invisible) {
```

Comment:
you should add this at the top of this inner method, with a return, and not indent the rest:
```
   if (!!field.attrs.invisible) {
                delete measures[name];
                return;
}

```

Reply:
also, do not forget ;

Reply:
also, invisible is a string which represent a python value. You should use pyUtils.py_eval to evaluate it (for example, if it is "0", it should not be considered invisible)